admin/timescale/updates/2024-05-28-add-pg_trgm-extension.sql

```diff
@@ -0,0 +1,2 @@
+CREATE EXTENSION IF NOT EXISTS "pg_trgm";
+ALTER DATABASE listenbrainz_ts SET pg_trgm.word_similarity_threshold = 0.1;
```

Comment:
Do you need to copy this configuration line anywhere else? I guess maybe create_extensuions.sql?
See for example to other create_extensions file:
https://github.com/metabrainz/listenbrainz-server/blob/4afff49fbf1d4b57883d9449d2c68714461dec41/admin/sql/create_extensions.sql#L2-L5

Reply:
I've already added this line in the create extension [file](https://github.com/metabrainz/listenbrainz-server/pull/2886/files#diff-745f27b3b8521e66884ca62ced68b7307d92f0f5ba37d8154b426f4335337a4cR3), and have added the second command in the [manage.py](https://github.com/metabrainz/listenbrainz-server/pull/2886/files#diff-da5414eec775a578bf2c026b63b632f4de08079068683a672bd9d810199cc48eR187-R191), so I guess this is enough. I haven't tested it, tho